config.go

```diff
@@ -502,6 +502,11 @@ type Config struct {
 	// HTTPHeaderTimeout is the maximum duration that the server will wait
 	// before timing out reading the headers of an HTTP request.
 	HTTPHeaderTimeout time.Duration `long:"http-header-timeout" description:"The maximum duration that the server will wait before timing out reading the headers of an HTTP request."`
+
+	// RepopulateMissingEdges is a value indicating whether missing edges of
+	// local channels should be re-added to the local view of the graph on
+	// startup.
+	RepopulateMissingEdges bool `long:"repopulate-missing-edges" description:"Repopulates any missing edges of local channels in the node's local view of the graph. This can help with 'edge not found' issues where LND will always select the default routing policy. Should be set to false after successful execution to avoid repopulating on every restart of lnd."`
```

Comment:
This looks more like a db migration for the affected nodes than a config to me.

Reply:
If the problem was permanently fixed in https://github.com/lightningnetwork/lnd/pull/7613 that could be a better solution. What do you think @ellemouton? Was the 'edge not found' issue reliably fixed by that PR?